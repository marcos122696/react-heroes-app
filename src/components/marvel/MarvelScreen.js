import React from 'react'
import { HeroList } from '../hero/HeroList'

export const MarvelScreen = () => {
  return (
    <div>
        <h1 className="animate__animated animate__hinge">
            MarvelScreen
        </h1>
        <hr />

        <HeroList publisher='Marvel Comics'/>
    </div>
  )
}
