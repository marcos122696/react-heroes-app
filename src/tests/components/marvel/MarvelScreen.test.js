import { shallow } from "enzyme";
import { MarvelScreen } from "../../../components/marvel/MarvelScreen";



describe('Pruebas en el componente <MarvelScreen />', () => {

    test('Debe de mostrarse correctamente', () => {

        const wrapper = shallow(<MarvelScreen />);
        expect(wrapper).toMatchSnapshot();
    });
});