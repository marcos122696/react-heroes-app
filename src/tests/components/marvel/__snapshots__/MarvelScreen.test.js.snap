// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <MarvelScreen /> Debe de mostrarse correctamente 1`] = `
<div>
  <h1
    className="animate__animated animate__hinge"
  >
    MarvelScreen
  </h1>
  <hr />
  <HeroList
    publisher="Marvel Comics"
  />
</div>
`;
